app/views/unattended/provisioning_templates/finish/windows_default_finish.erb

```diff
@@ -31,102 +33,108 @@ description: |
-      <% end %>
-    <% end %>
-  <% end %>
-
-  <% if host_param('localAdminAccountDisabled') %>
-    echo Disabling %tempAdminUser%
-    net user %tempAdminUser% %tempAdminUser% /active:no
-  <% end %>
-
-  <% if host_param('ansible_port') == 5985 or host_param('ansible_winrm_scheme') == 'http' %>
-    cmd /c winrm set winrm/config/service @{AllowUnencrypted="true"}
-  <% end %>
-
-  <% if host_param('ansible_winrm_transport') == 'basic' %>
-    cmd /c winrm set winrm/config/client/auth @{Basic="true"}
-    cmd /c winrm set winrm/config/service/auth @{Basic="true"}
-  <% end %>
-
-  <% if host_param('ansible_winrm_transport') == 'credssp' %>
-    cmd /c winrm set winrm/config/client/auth @{CredSSP="true"}
-    cmd /c winrm set winrm/config/service/auth @{CredSSP="true"}
-  <% end %>
-
-  <% if host_param('ansible_winrm_transport') == 'certificate' %>
-    cmd /c winrm set winrm/config/client/auth @{Certificate="true"}
-    cmd /c winrm set winrm/config/service/auth @{Certificate="true"}
-  <% end %>
-
-  <%= snippet 'Windows network' %>
-
-  <% if foreman_url('user_data') %>
-    echo execute user data script
-    IF EXIST c:\deploy\user_data.ps1 powershell.exe -OutputFormat text -command c:\deploy\user_data.ps1
-  <% end -%>
-
-  <% if puppet_enabled %>
-    echo Installing puppet
-    start /w "" msiexec /qn /i C:\extras\puppet.msi PUPPET_AGENT_STARTUP_MODE=Manual PUPPET_MASTER_SERVER=<%= host_puppet_server -%> PUPPET_AGENT_ACCOUNT_DOMAIN=<%= @host.domain -%> PUPPET_AGENT_ACCOUNT_USER=administrator PUPPET_AGENT_ACCOUNT_PASSWORD="<%= host_param('domainAdminAccountPasswd') -%>"
-    echo set puppet to auto start
-    sc config puppet start= auto
-    sc query puppet
-  <% end%>
-
-  <% if host_param('foremanDebug') != true %>
-
-    echo reboot in 15sec
-    start /b shutdown /r /t 15
-
-    echo Safely remove wimaging files
-    sdelete.exe -accepteula -p 2 -r c:\wimaging
-    sdelete.exe -accepteula -p 2 -r c:\minint
-    sdelete.exe -accepteula -p 2 c:\Windows\Panther\unattend.xml
-    sdelete.exe -accepteula -p 2 C:\Windows\Setup\Scripts\SetupComplete.cmd
-
-    echo Safely remove leftover directories
-    sdelete.exe -accepteula -p 2 -r c:\drivers
-    sdelete.exe -accepteula -p 2 -r c:\updates
-
-    echo Safely removing c:\deploy
-    cd /
-    sdelete.exe -accepteula -p 2 -r c:\deploy
-  <% end -%>
+echo Activating administrator
+net user administrator /active:yes
+<% end -%>
+
+<% if @host.pxe_build? -%>
+set ctr=0
+set nettimeout=10
+<% end -%>
+
+<%= snippet 'windows_network' %>
+
+<% if host_param('ntpServer') -%>
+echo Setting time server
+w32tm /config /manualpeerlist:<%= host_param('ntpServer') %> /syncfromflags:manual /update
+<% end -%>
+
+echo Syncing time
+w32tm /resync
+w32tm /resync
```

Comment:
Is it intentional that you call this twice?

Reply:
I removed the second call.